### PR TITLE
module standardization 

### PR DESCRIPTION
**Pull request by Rob1in**

### CODE REVIEW

**Comment by Rob1in** [comment]
I will convert it to not a draft after testing the requirements on linux/amd

File: src/distance.py [hunk 1]
```diff
@@ -1,20 +1,51 @@
...
-import torch.nn.functional as F
-from torch import Tensor
 import numpy as np
+import torch.nn.functional as F
+from torch.nn import CosineSimilarity
+
+
+def euclidean_distance(t1, t2):
+    """
+    Computes the Euclidean distance between two tensors using the L1 norm.
```
Comment (bhaney) [inline src/distance.py]: Euclidean usually means the L2 norm, not L1 -- the L1 norm is colloquially called the "manhattan" or "taxicab" norm

**Comment by Rob1in** [comment]
tested the build and the module on linux/amd

File: src/models/utils.py [hunk 2]
```diff
@@ -1,44 +1,81 @@
...
+
+    Raises:
+        ValueError: If the distance metric is not one of 'euclidean', 'euclidean_l2', or 'cosine'.
+    """
     if distance_metric == "euclidean":
         return 1.1
     if distance_metric == "euclidean_l2":
         return 1.1
-    elif distance_metric == "cosine":
+    if distance_metric == "cosine":
```
Comment (bhaney) [inline src/models/utils.py]: Not sure if it's a breaking change to change the name from euclidean to another name to make it more clear that it is the L1 norm, but if it isn't a breaking change, I would save change the name to "l1_norm" or something similar

**Review by bhaney — COMMENTED** [review]
nice job with the edits, just a question about a name -- if it's a breaking change to change the name of the distance function, then we shouldn't do it.

File: Makefile [hunk 3]
```diff
@@ -1,2 +1,8 @@
 module.tar.gz:
 	tar czf $@ --exclude='.DS_Store' *.sh .env src/*.py src/models/*.py src/models/checkpoints/*.pt src/models/checkpoints/*.onnx requirements.txt 
+
+test:
+	python3 -m pytest tests/*
+
+lint:
+	python3 -m pylint src --disable=W0719,C0114,W0201,W1203,E1101,W1203,E0611,R0902,R0913,E0611,E0001,R0914,R0903
```
Comment (bhaney) [inline Makefile]: C0114: Missing module docstring
W0201: Attribute defined outside init
W1203: Use % formatting in logging functions and pass the % parameters as arguments
E1101: Instance of 'x' has no 'y' member
E0611: No name 'x' in module 'y'
E0001: Syntax error
R0801: Similar lines in 'x' files

Remove these from the disable string, and if you do need to disable them, put the ignore right next to the code where you need it

**Review by bhaney — CHANGES_REQUESTED** [review]
Almost done, I think you just need to eliminate some of the ignored pylint codes, and then it should be good

**Review by bhaney — APPROVED** [review]
LGTM!

**Pull request merged**